core/arch/arm/plat-marvell/main.c

```diff
@@ -108,3 +114,21 @@ void console_init(void)
 #endif
 	register_serial_console(&console_data.chip);
 }
+
+#ifdef CFG_HW_UNQ_KEY_SUPPORT
+TEE_Result tee_otp_get_hw_unique_key(struct tee_hw_unique_key *hwkey)
+{
+	void *fuse_huk = NULL;
+
+	fuse_huk = phys_to_virt(PLAT_MARVELL_FUSF_FUSE_BASE +
+							PLAT_MARVELL_FUSF_HUK_OFFSET,
+							MEM_AREA_IO_SEC);
```

Comment:
Please fix alignment and assign directly. Also how about simplifying the names to make the code easier to read?
```suggestion
	void *huk = phys_to_virt(FUSF_FUSE_BASE + FUSF_HUK_OFFSET,
				 MEM_AREA_IO_SEC);
```